definitions/distributed_apps.md

```diff
@@ -5,9 +5,15 @@ category: concept
 ---
 ## Distributed Apps
 
-A distributed application is one where the logic required to perform a useful action is split amongst many programs that run as distinct processes and may be located on different computers, perhaps even in different data centres.
+### What it is
 
-An application running on one single computer is fragile - if that one computer fails for any reason, the application is totally unavailable. 
+A distributed application is an application where the functionality is broken down into multiple smaller independent parts. Distributed applications are usually composed of individual [microservices](https://github.com/cncf/glossary/blob/main/definitions/microservices.md) that handle different concerns within the broader application. In a cloud native environment the individual components typically run as [containers](https://github.com/cncf/glossary/blob/main/definitions/container.md) on a [cluster](https://github.com/cncf/glossary/blob/main/definitions/cluster.md). 
```

Comment:
```suggestion
A distributed application is an application where the functionality is broken down into multiple smaller independent parts. Distributed applications are usually composed of individual [microservices](https://github.com/cncf/glossary/blob/main/definitions/microservices.md) that are responsible for defined bounded contexts of functionality within the broader application. In a cloud native environment the individual components typically run as [containers](https://github.com/cncf/glossary/blob/main/definitions/container.md) on a [cluster](https://github.com/cncf/glossary/blob/main/definitions/cluster.md) or independent functions. 
```

Reply:
So I prefer the original language. I think different concerns is more accessible and not materially worse than bringing up bounded contexts which would also need to be defined.